sql-server-2014-alwayson-existing-vnet-and-ad/azuredeploy.json

```diff
@@ -0,0 +1,531 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+
+    "newStorageAccountNamePrefix": {
```

Comment:
Prefer to let user specify as they may be deploying multiple clusters on a common VNET and may have specific storage account naming requirements to track storage accounts per cluster.